package.json

```diff
@@ -19,6 +19,13 @@
   "peerDependencies": {
     "jsdoc": "3.6.x"
   },
+  "browserslist": [
+    "last 5 Chrome version",
```

Comment:
I don't think it causes any problems, but when selecting multiple versions, using `last 5 ... versions` instead of `last 5 ... version` would be more appropriate.